code/core/src/main/res/layout/push_template_filmstrip_carousel.xml

```diff
@@ -76,17 +81,17 @@
 
     <ImageButton
         android:id="@+id/leftImageButton"
-        android:layout_width="30dp"
-        android:layout_height="40dp"
-        android:layout_marginTop="130dp"
+        android:layout_width="15dp"
+        android:layout_height="20dp"
+        android:layout_centerVertical="true"
+        android:layout_alignParentLeft="true"
         android:background="@drawable/skipleft" />
 
     <ImageButton
         android:id="@+id/rightImageButton"
-        android:layout_width="30dp"
-        android:layout_height="40dp"
-        android:layout_marginTop="130dp"
+        android:layout_width="15dp"
+        android:layout_height="20dp"
```

Comment:
Would you be willing to move the dimensions into `dimens.xml` to ensure we have access and modify them from one place.

Reply:
yes, i will get it moved